packages/devtools_app/lib/src/service/editor/api_classes.dart

```diff
@@ -249,6 +257,27 @@ class DebugSessionStoppedEvent extends EditorEvent {
       };
 }
 
+class ThemeChangedEvent extends EditorEvent {
+  ThemeChangedEvent({required this.theme});
```

Comment:
can this be const?

Reply:
For this and other comments for this file: I tried to make the code added here consistent with the other events in this file, so I'm not sure if it's better to keep it this way, whether these suggestions would make sense for other events here, or if my additions should be moved to another place instead of in this file. I believe @DanTup set up this file so maybe he has advice.

Reply:
I took a deeper look at the code. Looks like this constructor cannot be const because of the non-const super constructor. 

WRT to consistency, fine to leave the named constructor over the factory constructor, but the ordering comment should be applied to follow the style guidelines: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#order-other-class-members-in-a-way-that-makes-sense

Feel free to adjust the getter order in other classes in this file where you see they violate the style guideline.

Reply:
> or if my additions should be moved to another place instead of in this file

This file was intended to be for strongly-typed access to the well-known editor services/events, so I do think this is the right place, and if there are any issues with the code/style here, it's probably better to update than put them elsewhere (and I'm happy to update the existing classes to be consistent with the changes you'd like to make if that makes sense).

Reply:
Thanks! I adjusted the final/getters order